src/libs/vehicle/ardupilot/ardupilot.ts

```diff
@@ -786,6 +791,24 @@ export abstract class ArduPilotVehicle<Modes> extends Vehicle.AbstractVehicle<Mo
     sendMavlinkMessage(paramSetMessage)
   }
 
+  /**
+   * Request the vehicle to send a pre-defined list of messages, on it's data stream
+   * Those are messages usually used by any Ardupilot vehicle
+   */
+  requestDefaultMessages(): void {
+    const frequencyHzToIntervalUs = (frequencyHz: number): number => {
+      return 1000000 / frequencyHz
+    }
+
+    const messagesWithIntervals = Object.entries(defaultMessageFrequency).map(([messageType, frequencyHz]) => {
+      return { id: getMAVLinkMessageId(messageType as MAVLinkType), intervalUs: frequencyHzToIntervalUs(frequencyHz) }
+    })
+
+    messagesWithIntervals.forEach((message) => {
+      this.sendCommandLong(MavCmd.MAV_CMD_SET_MESSAGE_INTERVAL, message.id, message.intervalUs)
```

Comment:
we should await here

Reply:
Should we?
This would delay a lot the request for all messages.
What's your concern with simultaneous requests?

Reply:
We should at least catch the exception here, otherwise the foreach will not run for all messages. Or use `Promise.allSettled`.

Reply:
Done.